app/polyfills/setPrototypeOf.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+
+// $FlowExpectedError: We want to add this polyfill for android
```

Comment:
Could you add a link to the issues on react-native repo as well as the one you created here?

Reply:
Added a link to our issue. I don't think there is a specific issue for it in RN, it is a well known fact that the android JS environment is missing in some areas. 

I also expect this to be a temporary fix, as it will be fixed within the fetch package in a future release.